packages/dataclass-editor/src/components/dataclass/master/DataClassMasterContent.css

```diff
@@ -2,3 +2,11 @@
   margin: var(--size-3);
   min-height: 0;
 }
+
+.cell-with-tooltip {
+  display: inline-block;
+}
+
+td .ui-tooltip-content {
+  color: var(--body);
+}
```

Comment:
why are these needed?

Reply:
`display: inline-block` adjusts the width of the cell content to center the tooltip. Without that, the tooltip looks like this:

![image](https://github.com/user-attachments/assets/e0a73261-4da1-433f-8e07-1e0f2e8445c1)

`color: var(--body)` overwrites the tooltip color. Otherwise, it seems to take the color from the cell, which when selected results in the tooltip content not being visible:

![image](https://github.com/user-attachments/assets/a69dcb6b-69c1-4f90-874e-b3062643d6c6)

Reply:
For the first I think you can simply use a <span> instead of a <div>.
For the second, I think this should be improved in the ui-components directly

Reply:
Going to make the tooltip improvement in ui-components.